openmc/mgxs/mgxs.py

```diff
@@ -1483,7 +1483,7 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
         if self.by_nuclide and nuclides == 'sum':
 
             # Use tally summation to sum across all nuclides
-            query_nuclides = self.get_nuclides()
+            query_nuclides = [nuclides]
```

Comment:
If nuclides is `sum` and `self.by_nuclide` is true, the `tally.summation()` method with nuclides set to `[sum]`. Then the `tally.summation()` is going to call `get_nuclide_index(nuclide)` where nuclide is `sum`. I'm not sure of the best design for this, but it looks like this case would fail. Is that correct?